initPlayerLocal.sqf

```diff
@@ -1,4 +1,5 @@
 //Reserved Slot
+//todo zeus ?
 if (str(player) in ["p7", "p7_1", "p7_2", "p7_3","p7_4","p7_5","p7_6","p7_7"]) Then
 {
    switch (getPlayerUID(player)) do
```

Comment:
Are the UIDs up to date ?

Reply:
ID are legacy ones. But the fact is p7, p7_1, etc, are nowhere in the mission file.

And by the way, it is not possible to design reserved slots by such a script. Run this script implies that the player has been able to connect, but if the server is full, he cannot even pass the lobby.